App/util/copy-photo.ts

```diff
@@ -0,0 +1,31 @@
+import { call } from 'redux-saga/effects'
+import RNFS from 'react-native-fs'
+
+import AppConfig from '../Config/app-config'
+
+interface PathContaining {
+  path: string
+}
+
+export default function* copyPhoto<T extends PathContaining>(obj: T) {
```

Comment:
Check it out @undercase -- Generics FTW!

Reply:
Love this!